substrate/client/cli/src/error.rs

```diff
@@ -78,6 +80,19 @@ pub enum Error {
 
 	#[error(transparent)]
 	GlobalLoggerError(#[from] sc_tracing::logging::Error),
+
+	#[error(
+		"Starting an authorithy without network key in {0}.
+		\n This is not a safe operation because the old identity still lives in the dht for 36 hours.
+		\n Because of it your node might suffer from not being properly connected to other nodes for validation purposes.
+		\n If it is the first time running your node you could use one of the following methods:
+		\n 1. [Preferred] Separately generate the key with: polkadot key generate-node-key --default-base-path
+		\n 2. [Preferred] Separately generate the key with: polkadot key generate-node-key --base-path <YOUR_BASE_PATH>
+		\n 3. [Preferred] Separately generate the key with: polkadot key generate-node-key --file <YOUR_PATH_TO_NODE_KEY>
```

Comment:
```suggestion
		\n 1. [Preferred] Separately generate the key with: node key generate-node-key --default-base-path
		\n 2. [Preferred] Separately generate the key with: node key generate-node-key --base-path <YOUR_BASE_PATH>
		\n 3. [Preferred] Separately generate the key with: node key generate-node-key --file <YOUR_PATH_TO_NODE_KEY>
```

This is not only used by `polkadot`.